Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -624,12 +626,20 @@ void MediaPlayerPrivateGStreamer::seek(float time)
             endTime = clockTime;
         }
 
+        if (isMediaSource()) {
```

Comment:
For if blocks with only one statement please don't use curly braces. Applies in some other changes of this patch :)